infrastructure/cdn-in-a-box/traffic_ops/run-db.sh

```diff
@@ -18,5 +18,11 @@
 
 set-dns.sh
 insert-self-into-dns.sh
+set -o monitor #enable job control
+/docker-entrypoint.sh postgres &
 
-exec /docker-entrypoint.sh postgres
+source /to-access.sh
+# Enroll with traffic ops
+TO_URL="https://$TO_FQDN:$TO_PORT"
+to-enroll db ALL '' 5432 5432 || (while true; do echo "enroll failed."; sleep 3 ; done)
```

Comment:
I just wanna call this out here; I know this is "standard" in that it's what our other containers do, but we should seriously consider just failing if enrollment fails on non-optional containers.